apps-rendering/src/client/liveblog.ts

```diff
@@ -0,0 +1,23 @@
+import { ResultKind } from '@guardian/types';
+import { formatLocalTimeDateTz } from 'date';
+import { dateParser, parse } from 'parser';
+import { logger } from '../logger';
+
+function lastUpdatedDates(): void {
+	Array.from(document.getElementsByClassName('js-last-updated-time')).forEach(
```

Comment:
Are we selecting using `js-` based on backwards compatibility reasons? This is a legacy method for selecting elements.

Reply:
This was a suggestion from Jamie to follow similar patterns already implemented. We apply a className to the component with the prefix `js-` to show that it is used as part of the javascript function. It's used as an indicator for future changes that this className is important.

Previously I was doing a query selector to get all elements by data-attribute, but we decided to change. Are you suggesting I revert back to selecting by data attribute?

Reply:
I can't speak to how AR selects elements but from a DCR perspective, the `js-` thing is a Frontend pattern. It is present in a few isolated locations in DCR but only where we have contracts with external libs, like commercial.

In DCR we use data attributes for selecting elements. See `data-gu-*` or `data-ignore-*` but even then this is done rarely. Because we're using React and Emotion we rarely need global selectors.

Because this use case is proposing a global action I think this should be broken out into its own PR and an approach considered there. You are not going to just want to change this one date value to be local, you will inevitably want to change all the dates on the page (if you don't you will have dates showing in different timezones, which is bad).

[This PR](https://github.com/guardian/dotcom-rendering/pull/3704) shows how we broke out a script to manage relative time, which could, perhaps, offer a suggestion for a solution to local time.

When choosing any contract we should be mindful of the history and try to locate any decisions that were taken in the past. I'm sure this has been discussed before!

Reply:
We spoke about this with Jamie - the pattern for prefixing the class name with `js-` serves as an indicator of this contract and is used elsewhere in apps-rendering and The Guardian. We decided to keep as it is for this case.